lib/http.js

```diff
@@ -70,12 +75,12 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
       const {account} = record;
       actor = await brAccount.getCapabilities({id: account.id});
       const user = {account, actor};
-      req.logIn(user, err => {
-        if(err) {
-          return next(err);
-        }
-        res.json(account);
-      });
+
+      const login = promisify(req.logIn).bind(req);
```

Comment:
BTW shouldn't we be calling `promisify(req.logIn.bind(req))` as your bind is being called on a promise and not the original function. I might be wrong here.